release_notes.md

```diff
@@ -4,9 +4,10 @@
 - My change description (#PR)
 -->
 - Update Java Worker Version to [2.12.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.12.0)
-- Update Python Worker Version to [4.14.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.14.0)
 - Update protobuf to [v1.9.0-protofile](https://github.com/Azure/azure-functions-language-worker-protobuf/releases/tag/v1.9.0-protofile)
```

Comment:
I see this in [4.22.0](https://github.com/Azure/azure-functions-host/releases/tag/v4.22.0), so it should not be here